src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/Utilities.cs

```diff
@@ -15,6 +15,7 @@
 
 namespace System.Linq.Expressions.Interpreter
 {
+#if FEATURE_MAKE_RUN_METHODS
```

Comment:
Yes, I did not want to throw code like this away since it looked like some perf improvements that were disabled (or unfinished). And the likely reason is that it could require platform-specific support. 
I.E. may not be available/profitable on .Net Native, but otherwise could be a viable improvement.